include/pinocchio/algorithm/regressor.hpp

```diff
@@ -372,6 +372,46 @@ namespace pinocchio
     const ModelTpl<Scalar, Options, JointCollectionTpl> & model,
     DataTpl<Scalar, Options, JointCollectionTpl> & data,
     const Eigen::MatrixBase<ConfigVectorType> & q);
+
+  /// \brief Computes the indirect regressors Y_Hqd and Y_CTqd
+  /// of momentum and transposed coriolis matrix times velocity.
+  ///
+  /// These regressors are such that:
+  /// \f$ p = Y_Hqd(q, v) \pi$ and \f$ C^T v = Y_CTqd(q, v) \pi \f$
```

Comment:
Just a very small nitpicking: does this include the gravity vector?
Like, should this be` \f$ C^T v - g = Y_CTqd(q, v) \pi \f$`?

Reply:
No, it does not include gravity. In sysid example I provide a complete comparison and matching of these terms: https://github.com/stack-of-tasks/pinocchio/pull/2411/files#diff-de29e11f15faef229b4e025cc6c9f09af5d32c5268973e47484fc0b08afa6d77R154-R165